src/devices/Camera/tests/TestCamera/CommandOptionsBuilderTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Iot.Device.Camera.Settings;
+
+namespace TestCamera;
+
+/// <summary>
+/// Tests for the CommandOptionsBuilder class
+/// </summary>
+public class CommandOptionsBuilderTests
+{
+    /// <summary>
+    /// Test the command line arguments
+    /// </summary>
+    [Fact]
+    public Task TestCommandLineArguments1()
+    {
+        // repeating the options will NOT repeat the arguments
+        var builder = new CommandOptionsBuilder()
+            .WithContinuousStreaming()
+            .WithContinuousStreaming()
+            .WithH264VideoOptions("baseline", "4", 15)
+            .WithResolution(640, 480);
+
+        var args = builder.GetArguments();
+        Assert.Equal(10, args.Length);
```

Comment:
why is it 10?

Reply:
10 is the number of arguments added to the command line. I added the following comment
```csharp
// new CommandOptionsBuilder() => 1
// .WithContinuousStreaming() => 2
// .WithH264VideoOptions(...) => 5
// .WithResolution(...) => 2
```

Reply:
Can you please then add a little comment in the code about this magic number?